OrbitVulkanLayer/resources/VkLayer_Orbit_implicit.json

```diff
@@ -0,0 +1,18 @@
+{ 
+    "file_format_version" : "1.1.0", 
+    "layer" : { 
+        "name": "ORBIT_VK_LAYER", 
+        "type": "GLOBAL", 
+        "library_path": "./libOrbitVulkanLayer.so",
+        "api_version": "1.1.77", 
```

Comment:
What determines that API version we have to use or we can use? (Why exactly 1.1.77?)

Reply:
TBH, that was the version that GpuVoyeur was using.
The version basically states which API that layer is using. As all the methods we are using that are not in GpuVoyeur, were present in 1.1, I just sticked with that.

Reply:
Seems it is then fine to keep as is, and we can adjust if needed.